### PR TITLE
Partial translation in French

### PR DESCRIPTION
**Pull request by espinosa**
My first contribution to this fantastic project I was waiting from a long time. Thanks!

### CODE REVIEW

File: CliClient/locales/fr_FR.po [hunk 1]
```diff
@@ -575,6 +577,8 @@ msgstr "Afficher un résumé des notes et carnets."
 msgid ""
 "To retry decryption of these items. Run `e2ee decrypt --retry-failed-items`"
 msgstr ""
+"Exécutez `e2ee decrypt --retry-failed-items` pour tenter à nouveau le "
+"décryptage"
```
Comment (laurent22) [inline CliClient/locales/fr_FR.po:581]: déchiffrage

**Comment by tessus** [comment]
@laurent22 we have a problem with CI again:

```
1) integration_ForwardBackwardNoteHistory should save history when navigating through notebooks
  Message:
    Expected undefined to equal [  ].
  Stack:
    Error: Expected undefined to equal [  ].
        at <Jasmine>
        at asyncTest (/Users/travis/build/laurent22/joplin/CliClient/tests-build/feature_ForwardBackwardNoteHistory.js:97:38)
        at <Jasmine>

2) integration_ForwardBackwardNoteHistory should save history when navigating through notes
  Message:
    Expected undefined to equal [  ].
  Stack:
    Error: Expected undefined to equal [  ].
        at <Jasmine>
        at asyncTest (/Users/travis/build/laurent22/joplin/CliClient/tests-build/feature_ForwardBackwardNoteHistory.js:41:38)
        at <Jasmine>
```

**Comment by tessus** [comment]
@espinosa thanks for the update.

**Comment by espinosa** [comment]
@tessus You are welcome. Glad to help.

**Pull request merged**